spec/models/wishlist_spec.rb

```diff
@@ -1,6 +1,13 @@
 require "rails_helper"
 
 RSpec.describe Wishlist, type: :model do
```

Comment:
No problem, I removed it from both specs with the newest commit.
Please +1 if you think it's ready.